terraform/aws/modules/vpc-setup/vpc.tf

```diff
@@ -0,0 +1,24 @@
+
+resource "aws_vpc" "vpc_creation" {
+  for_each = toset(var.vpc_cidrs)
+
+  cidr_block                       = each.value
+  enable_dns_hostnames             = var.enable_dns_hostnames
+  tags = merge(
+    {
+      "Name" = format("%s-%s", var.name, join("", split(".", split("/", each.value)[0]))),
+      "Available" = "true"
```

Comment:
I just realized that it would be nice to have one more tag for each VPC. Something like `Owner` which the provisioner could put the cluster ID in as the value when it takes the VPC. That would help us keep track of what AWS resources are owned by which clusters.

Can that be easily added/updated now?

Reply:
Yeap I will add it Gabe :)